doc/releases/release-notes-3.1.rst

```diff
@@ -1123,3 +1123,799 @@ Issue Related Items
+* :github:`42960` - Bluetooth: Audio: Codec config parsing and documentation
+* :github:`42953` - it8xxx2_evb: Test in tests/kernel/timer/timer_api fail.
+* :github:`42943` - [backport v2.7-branch] backport of #42457 failed
+* :github:`42940` - Please add zsock_getpeername
+* :github:`42928` - CSIS: Invalid usage of bt_conn_auth_cb callbacks
+* :github:`42919` - [backport v2.6-branch] backport of #42422 failed
+* :github:`42915` - [backport v2.7-branch] backport of #41716 failed
+* :github:`42888` - Bluetooth: Controller: Extended Advertising - Advertising Privacy Support
+* :github:`42881` - Arduino due missing 'arduino_i2c' alias.
+* :github:`42877` -  k_cycle_get_32 returns 0 on start-up on native_posix
+* :github:`42874` - ehl_crb: samples/kernel/metairq_dispatch fails when it is run multiple times
+* :github:`42870` - Build error due to minimal libc qsort callback cast
+* :github:`42865` - openocd configurations missing for stm32mp157c_dk2 board
+* :github:`42857` - sam_e70b_xplained: failed to run test cases tests/net/npf and tests/net/bridge
+* :github:`42856` - Bluetooth: BAP: Unicast client sample cannot connect
+* :github:`42854` - k_busy_wait() never returns when called - litex vexriscv soc and cpu on xilinx ac701 board
+* :github:`42851` - it8xxx2_evb: Mutlitple tests in tests/kernel/contex fail.
+* :github:`42850` - CONFIG items disappeared in zephyr-3.0-rc3
+* :github:`42848` - it8xxx2_evb: Test in /tests/subsys/cpp/libcxx fail.
+* :github:`42847` - it8xxx2_evb: Multiple tests in tests/subsys/portability/cmsis_rtos_v2 fail.
+* :github:`42831` - Do the atomic* functions require protection from optimization?
+* :github:`42829` - GATT: bt_gatt_is_subscribed does not work as expected when called from bt_conn_cb->connected
+* :github:`42825` - MQTT client disconnection (EAGAIN) on publish with big payload
+* :github:`42817` - ADC on ST Nucleo H743ZI board with DMA
+* :github:`42800` - gptp_mi neighbor_prop_delay is not included in sync_receipt_time calculation due cast from double to uint64_t
+* :github:`42799` - gptp correction field in sync follow up message does not have correct endianness
+* :github:`42774` - pinctrl-0 issue in device tree building
+* :github:`42723` - tests: kernel.condvar: child thread is not running
+* :github:`42702` - upsquared: drivers.counter.cmos.seconds_rate is failing with busted maximum bound when run multiple times
+* :github:`42685` - Socket echo server sample code not working in Litex Vexriscv cpu (Xilinx AC701 board)
+* :github:`42680` - Missing bt_conn_(un)ref for LE Audio and tests
+* :github:`42672` - [backport v2.7-branch] backport of #38603 failed
+* :github:`42631` - v2.7: Unable to identify owner of net_mgmt_lock easily
+* :github:`42622` - [ 2.7.1 ] pm: pm_device structure bigger than nessecary when PM_DEVICE_RUNTIME not set
+* :github:`42599` - tests: kernel: mem_protect: mem_protect fails after reset on stm32wb55 nucleo
+* :github:`42588` - lsm6dso
```

Comment:
again missing some details here

Reply:
That's the reporter's fault; I'm OK with not fixing it for these release notes.